test_compileall2.py

```diff
@@ -352,6 +352,34 @@ def test_multiple_optimization_levels(self):
                 except Exception:
                     pass
 
+    @support.skip_unless_symlink
+    def test_ignore_symlink_destination(self):
+        # Create folders for allowed files, symlinks and prohibited area
+        allowed_path_parts = ["test", "dir", "allowed"]
+        symlinks_path_parts = ["test", "dir", "symlinks"]
+        prohibited_path_parts = ["test", "dir", "prohibited"]
+        allowed_path = os.path.join(self.directory, *allowed_path_parts)
+        symlinks_path = os.path.join(self.directory, *symlinks_path_parts)
+        prohibited_path = os.path.join(self.directory, *prohibited_path_parts)
```

Comment:
Why don'y you get rid fo the parts variables?

```suggestion
        prohibited_path = os.path.join(self.directory, "test", "dir", "prohibited")
```

Reply:
Fixed. Okay now?